### PR TITLE
Upgrade eslint + move to eslint-config-airbnb-base

### PR DESCRIPTION
**Pull request by hipstersmoothie**
Moving to standardized tooling and configs. Turned off rules that clashed with how the code is currently written.

This PR should be merged after #1076

### CODE REVIEW

**Comment by leerob** [comment]
This can be updated now after Prettier has been merged!

**Comment by leerob** [comment]
Nice job here 😄 🚀

File: prelude/bootstrap.js [hunk 1]
```diff
@@ -18,16 +20,16 @@
...
-var isRootPath = common.isRootPath;
-var normalizePath = common.normalizePath;
-var insideSnapshot = common.insideSnapshot;
-var stripSnapshot = common.stripSnapshot;
-var removeUplevels = common.removeUplevels;
+var { isRootPath } = common;
+var { normalizePath } = common;
+var { insideSnapshot } = common;
+var { stripSnapshot } = common;
+var { removeUplevels } = common;
```
Comment (robertsLando) [inline prelude/bootstrap.js:32]: This could become `const { STORE_GLOB, STORE_CONTENT, ...} = common`

**Review by robertsLando — CHANGES_REQUESTED** [review]
Nice work here! I think you forgot to delete `test/test-46-input-output-targets-many-6/test-output.exe-linux`

**Comment by hipstersmoothie** [comment]
I have addressed all code review comments 🚀

**Review by robertsLando — APPROVED** [review]
LGTM

**Comment by leerob** [comment]
Assuming the tests pass, let's merge!

**Comment by robertsLando** [comment]
Tests are so slow, takes around 20/30 minutes :(

**Comment by leerob** [comment]
It's important they pass with a change this large 😄

**Comment by leerob** [comment]
Totally agree. Luckily, they passed!

**Pull request merged**